DotnetViewComponents/ViewComponents/InsetTextViewComponent.cs

```diff
@@ -0,0 +1,18 @@
+namespace DotnetViewComponents.ViewComponents
+{
+    using System.Collections.Generic;
+    using Microsoft.AspNetCore.Mvc;
+    using DotnetViewComponents.ViewModels;
+
+    public class InsetTextViewComponent : ViewComponent
+    {
+        public IViewComponentResult Invoke(
+            string text,
+            string cssClass = null
```

Comment:
Why are we including this parameter? Do we ever alter the class of an inset text div?